README.md

```diff
@@ -2,6 +2,12 @@
 
 A quickstart for the trader agent for AI prediction markets on Gnosis at https://github.com/valory-xyz/trader
 
+## Compatible Systems
+
+- Windows 10/11: WSL2 / Git BASH
+- Mac ARM / Intel
+- Linux
```

Comment:
```suggestion
- Ubuntu 22.04 LTS
```

Reply:
The reason that I suggest Linux is that it has been confirmed to work in many Linux distributions/platforms: Ubuntu 22.04 and 20.x, Fedora, AWS Linux 2, etc